src/CodeGen_LLVM.cpp

```diff
@@ -287,8 +287,7 @@ void CodeGen_LLVM::initialize_llvm() {
 
 void CodeGen_LLVM::init_context() {
     // Ensure our IRBuilder is using the current context.
-    delete builder;
-    builder = new IRBuilder<>(*context);
+    builder.reset(new IRBuilder<>(*context));
```

Comment:
Suggestion: use `std::make_unique` instead? (Your code isn't wrong but as of C++14 make_unique is in theory the 'preferred' way to do this)

Reply:
Done